.gitlab-ci.yml

```diff
@@ -77,7 +77,7 @@ after_script:
 
     - echo 'start:coq.build'
     - make -j "$NJOBS" byte
-    - make -j "$NJOBS" world $EXTRA_TARGET
+    - make COQUSERFLAGS="${COQ_USER_FLAGS}" -j "$NJOBS" world $EXTRA_TARGET
```

Comment:
Remove the configure change, do `COQUSERFLAGS?=` in makefile.build before using the variable, and do nothing here (pass as environment variable).

Reply:
Done, thanks for the suggestion!